packages/jest-core/src/SearchSource.ts

```diff
@@ -69,29 +51,45 @@ const toTests = (context: Context, tests: Array<Config.Path>) =>
 
 export default class SearchSource {
   private _context: Context;
-  private _rootPattern: RegExp;
-  private _testIgnorePattern: RegExp | null;
   private _testPathCases: TestPathCases;
```

Comment:
while we're refactoring stuff, this could be initialized here instead in constructor

Reply:
What could be? You're looking at removed lines. :)

Reply:
I meant testPathCases

Reply:
Done.